torch/_C/_distributed_c10d.pyi

```diff
@@ -209,6 +209,21 @@ class ProcessGroup:
         op=ReduceOp.SUM,
     ) -> Work: ...
     @overload
+    def _reduce(
+        self,
+        output_tensors: List[Tensor],
+        input_tensors: List[Tensor],
+        opts=ReduceOptions(),
+    ) -> Work: ...
+    @overload
+    def _reduce(
+        self,
+        output_tensor: Tensor,
+        input_tensor: Tensor,
+        root: int,
+        op=ReduceOp.SUM,
+    ) -> Work: ...
+    @overload
```

Comment:
**For your use case**, do you need both API signatures or just need one? I tend to think just need the Tensor version at the Python level.
Also, for clarity of the API name, you can name it `_reduce_oop`.

Reply:
I have removed the API signature that is not needed and renamed `_reduce` to `_reduce_oop` everywhere.